network-api/networkapi/wagtailpages/migrations/0016_merge_20201117_1917.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.16 on 2020-11-17 19:17
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailpages', '0015_auto_20201110_1957'),
+        ('wagtailpages', '0015_buyersguidepage'),
+    ]
+
+    operations = [
+    ]
```

Comment:
we clearly don't needs this migration =D

(just delete it rename the 0017 to 0016)